### ISSUE TITLE
Teleport requires hold/stop option is bugged when playing Pugna

### ISSUE THREAD

**Issue by luckytaurus11** [ValveSoftware/Dota2-Gameplay#19090]
### Ability name

Nether Blast

### Description

Granted I only tested about 20 heroes but so far it seems that only Pugna is bugged. Casting any ground-targeted ability while tping will cancel your tp despite having that option enabled. Works with either your Nether Blast or your Nether Ward. 

This happened to me in game and it continues to happen in demo mode.

Game ID: 7774423344

Time stamped example: 18:45

### Example Match ID (and possibly Timestamp)

7774423344 at time 18:45

**Comment by qudesn1q** [comment]
Yes. The option to cancel teleport using a separate key does not work on Pugna. And this still hasn't been fixed.
